src/software/constants.h

```diff
@@ -8,22 +8,6 @@
 // considered as "gone" and no longer reported.
 static constexpr unsigned int ROBOT_DEBOUNCE_DURATION_MILLISECONDS = 200;
 
-
-static constexpr unsigned int MAX_SIMULATOR_MULTICAST_CHANNELS = 16;
```

Comment:
do we need both shared/constants and software/constants?

Reply:
No we definetly don't, the work is tracked here

https://github.com/UBC-Thunderbots/Software/issues/2627